server/app.py

```diff
@@ -12,4 +12,77 @@
 app.register_blueprint(ping_handler)
 
 db = SQLAlchemy(app)
-migrate = Migrate(app, db)
\ No newline at end of file
+migrate = Migrate(app, db)
+
+from models import List, Product
+
+@app.route('/create_list', methods=['POST'])
+def createList():
+    if request.method == 'POST':
+        body = json.loads(request.get_data())
+        list_item = List(body['product_type'])
+        db.create_all()
+        db.session.add(list_item)
+        try:
+            db.session.commit()
+        except:
+            return jsonify({'response': "{} already exists in the database".format(body['product_type'])}), 400
```

Comment:
There's other reasons this could fail, right? Maybe we can except Exception as e: and return e.message

Reply:
e.message was deprecated, instead I used e.__cause__, hope that's alright